sdk/extensions/wcf/Microsoft.WCF.Azure.StorageQueues/src/Microsoft.WCF.Azure.StorageQueues.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <RequiredTargetFrameworks>net6.0</RequiredTargetFrameworks>
```

Comment:
These should use the standard properties from the central engineering system, though I agree that it makes sense to override the required target frameworks given the ASP.NET focus.   We should add a comment about that.

Reply:
Fair enough; the override seems appropriate then.   I'd remove L4 in that case and just use what you have hardcoded on L5.